src/models/spaces/organisms/organisms-row.ts

```diff
@@ -30,36 +30,52 @@ export const OrganismsRowModel = types
     hoveredOrganelle: types.maybe(Organelle),
     selectedOrganelle: types.maybe(Organelle),
     mode: types.optional(Mode, "normal"),
-    assayedOrganelle: types.maybe(Organelle),
+    assayedOrganelles: types.array(Organelle),
     zoomLevel: types.optional(ZoomLevel, "organism"),
     showProteinDNA: false,
     showProteinAminoAcidsOnProtein: false,
     proteinSliderStartPercent: 0
   })
   .views(self => ({
     get currentData(): ChartDataModelType {
-      const chartDataSets = [];
-      const points: DataPointType[] = [];
-      const organelle = self.assayedOrganelle;
-      if (organelle) {
-        kSubstanceNames.forEach((substance) => {
+      const chartDataSets: any[] = [];
+      const organelleLabels: string[] = [];
+
+      self.assayedOrganelles.forEach((organelle) => {
+        organelleLabels.push(organelle);
+      });
+      const colorMaps: any = { pheomelanin: "#f4ce83",
+                               signalProtein: "#d88bff",
+                               eumelanin: "#795423",
+                               hormone: "#0adbd7" };
+      let stackIterator = 0;
+      kSubstanceNames.forEach((substance) => {
+        const substanceName = substance === "signalProtein" ? "signal protein" : substance;
+        const substanceColor: string = colorMaps[substance];
+        const points: DataPointType[] = [];
+        self.assayedOrganelles.forEach((organelle) => {
           if (!self.organismsMouse) {
             return;
           }
           const substanceValue = self.organismsMouse.getSubstanceValue(organelle, substance);
-          points.push(DataPoint.create({ a1: substanceValue, a2: 0, label: `${organelle} ${substance}` }));
+          points.push(DataPoint.create({ a1: substanceValue, a2: 0, label: `${organelle} ${substanceName}` }));
         });
-      }
-
-      chartDataSets.push(ChartDataSetModel.create({
-        name: self.assayedOrganelle || "",
-        dataPoints: points,
-        pointColors: ["#f4ce83", "#d88bff", "#795423", "#0adbd7"]
-      }));
+        const stackNum = "Stack " + stackIterator;
+        chartDataSets.push(ChartDataSetModel.create({
+          name: substanceName,
+          dataPoints: points,
+          color: substanceColor,
+          fixedMaxA2: 800,
+          fixedMaxA1: 800,
```

Comment:
I set the max to 800 which was used in the previous version.

Reply:
Looks good to me!

Reply:
👍